### PR TITLE
SU form

### PR DESCRIPTION
**Pull request by Srjnnnn**
The page looks like the original page.
Needs a review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,74 @@
...
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```
Comment (HADDADSOHAIB) [inline README.md]: Remove this if you are not using those sections

File: index.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                            <div>
+                                E-mail
+                            </div> 
+                            <input type="email" name="email" /> 
+                        </label>
+                        <label>
+                            <div>
+                                Phone
+                            </div>
+                            <input type="text" name="number" />
```
Comment (HADDADSOHAIB) [inline index.html]: The type should be tel:
- [ ] It has an `<input>` of type `tel` and a label `Phone` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))

File: index.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+                        </label>
+                        <label>
+                            <div>
+                                Password
+                            </div>
+                            <input type="password" name="password" /> 
+                        </label> 
+                    </div>
+                    <div class="form-input-button">
+                        <input type="button" value="Sign in" />
```
Comment (HADDADSOHAIB) [inline index.html]: The form should have a button with type submit:
- [ ] It has a `<submit>` button that matches the style of the original page (Blue background, white text) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Change Required :recycle: 

### Hello @eypsrcnuygr and @AlduLonghi    :handshake: 

### Good job so far :clap:  your project is looking good :medal_sports: 

### there are some changes to get the project approved :point_down:
- I think your readme could be improved, the readme file should be descriptive:
Screenshot from 2020-06-05 01-53-51
It will be better if you put a name for the project (ex: sign up page clone, mint sign up clone ...)
Screenshot from 2020-06-05 01-54-05
If you are not using a section, take it out, the readme file should stay clean.
Screenshot from 2020-06-05 01-55-15
Your pull request description could also be improved, always remember to be descriptive, take a look at this link to understand why a github repository should look professional:
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/.
- I don't see any linter configuration files, it is required to set up that in your project, please read this link again and add the required files: https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/
- Your project did not passe the validators:
- [ ] Check HTML markup with this validator https://validator.w3.org/#validate_by_upload. (Students were asked to use this one, so in most cases their code should be already validated) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/validation_requirement.png) from Pathwright)
Screenshot from 2020-06-05 02-15-49
Always passe your code through validators before pull requests.

**Comment by Srjnnnn** [comment]
We've done all the warnings that you gave to us.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+                <a href="www.facebook.com"><p>Do you forget your password?</p></a>
+            </div>
+            <div class="inputbutton">
+                <input type="button" name="signin" value="Sign in"/>
+            </div>   
+        </div>
+    </header>
+    <section class="Main-header">
+        <div>
+            <h1>intuit</h1>
```
Comment (peterrobert) [inline index.html]: The intuit logo is supposed to be an image, not text.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+                            <input type="password" name="password" /> 
+                        </label> 
+                    </div>
+                    <div class="form-input-button">
+                        <input type="submit" value="Sign in" />
+                    </div>
+                    <div class="form-end">
+                        <p><i>By clicking Create account, you agree to our
+                            <a href="https://www.w3schools.com">Terms</a> and have read and acknowledge our <a href="https://www.w3schools.com">Privacy Statement</a>.</i></p>
+                    </div>
```
Comment (peterrobert) [inline index.html:75]: This text should be centered. 
your project :arrow_down: 
yours1
the clone :arrow_double_down: 
yours2

File: index.html [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+                        <div>
+                            Password
+                        </div>
+                        <label>
+                            <input type="password" name="password" /> 
+                        </label> 
+                    </div>
+                    <div class="form-input-button">
+                        <input type="submit" value="Sign in" />
+                    </div>
```
Comment (peterrobert) [inline index.html]: The button displays create an account not sign in

clone :arrow_down: 
yours2

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :stop_sign: 
Great work so far :clap: However, there are some changes that are required before I approve this project

- [x] I don't see your Linters config files. This has to be added to the project since its a requirement.

Below there some more improvements to be made.

**If you have a question concerning this review feel free to contact me** @peterrobert

### HAPPY CODING

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hello team :wave:
-
You have done a splendid job with the changes from the TSEs! :tada: Keep it up.
### [The clone](https://raw.githack.com/AlduLonghi/SignUpPage/SUform/index.html) shows good use of HTML forms. :100: 

A few required changes: Note that for this project, we have to make sure nothing is messed up. Please see the difference in the source page and the clone below: 

##
|[Source Page](https://accounts.intuit.com/signup.html)|[Clone Page](https://raw.githack.com/AlduLonghi/SignUpPage/SUform/index.html)|
|:---:|:---:|
|source|clone|
- Let's improve on the general styling of the form page.
- - [ ] The top right Signup section. Observer the differences.
- -  [ ] The Forms main logo and the small logos as well.
- - [ ] The Footer of the form section.
- - [ ] And lastly the page's fonts and font colours.
- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Change Required :recycle: 

### Hello @eypsrcnuygr and @AlduLonghi    :handshake: 

### :medal_sports: You are doing a great job :+1:  nice work with the logo, it looks like the original.

### Unfortunately, I can't approve the project if the elements of the page are not placed in the correct way, you need always to check your page before you submit.
- Your project:
Screenshot from 2020-06-08 10-14-43
- As you can see the password field need to be fixed.
- Great job so far, I am sure you will fix it. Good luck :1st_place_medal:

**Review by HADDADSOHAIB — APPROVED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Project Approved :100: 

### Hello @eypsrcnuygr and @AlduLonghi    :handshake: 
### Congratulations your project is approved,  you can move on and merge into master.
### Good job.

**Comment by Srjnnnn** [comment]
Thanks

**Pull request merged**